workspace-manager-clienttests/src/main/resources/servers/workspace-ichang.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "workspace-ichang",
+  "description": "Workspace Manager Resiliency Environment",
```

Comment:
nit: ichang personal development environment?

Reply:
The `terra-wsmtest` environment has been set up for resiliency tests. I have switched the test over to `wsmtest` namespace.

Reply:
ok that makes sense. My comment here was just about updating the description field to note that it's your personal development environment. (e.g. "Workspace Manager ichang personal environment").

Reply:
Updated description field.